src/clock/mumeclock.cpp

```diff
@@ -410,3 +410,12 @@ MumeClock::DawnDusk MumeClock::getDawnDusk(const int month)
     const auto m = static_cast<uint32_t>(month);
     return DawnDusk{s_dawnHour.at(m), s_duskHour.at(m)};
 }
+
+int MumeClock::getMumeMonth(const QString &monthName)
```

Comment:
Should we have this return the enumerated value instead?

Reply:
I'll have a look at it.

Reply:
This might be changing the entire data structure and a bunch of other code. If so, it's probably best saved for a future PR.